### DISCUSSION TITLE
Change host for Remix App Server

### DISCUSSION THREAD

**Discussion by rasmusfiskerbang** [remix-run/remix#4621] [Q&A]
Hi

How do I change host for Remix App Server. It insists on using localhost and I need to change it to 0.0.0.0
I can set HOST='0.0.0.0' but that only seems to change the 'secondary' host, it is still served on localhost as well.

I'm trying to deploy on render, and they require host to be 0.0.0.0, otherwise i get a bad gateway in the health checks

Thanks!

**Comment by sergiodxa** [comment]
From the code (see https://github.com/remix-run/remix/blob/d6d0e2d9652a5aae6f95b6c73f3dccf71e91a282/packages/remix-serve/cli.ts#L44) the only way is using the HOST env variable, that env is passed to Express `server.listen` method directly, so maybe it's Express the one serving it on localhost too.
